python/tests/test.py

```diff
@@ -307,12 +312,14 @@ def test_get_reports_from_population(self):
         keys.sort()
         self.assertEqual(keys, [(13, 30), (13, 30), (13, 31), (13, 31), (13, 32), (14, 32), (14, 33), (14, 33), (14, 34), (14, 34)])
 
-        self.assertEqual(len(sel.times), 3)  # Number of timestamp (0.8, 1.0 and 1.2)
-        sel = self.test_obj['All'].get(tstart=5., tstop=-1)  # tstart out of range
-        self.assertEqual(len(sel.ids), 0)
-        sel = self.test_obj['All'].get(tstart=3., tstop=3.)
-        self.assertEqual(len(sel.ids), 100)
+        self.assertEqual(len(self.test_obj['All'].get(node_ids=[]).data), 0)
+        self.assertEqual(len(self.test_obj['All'].get(node_ids=[]).times), 0)
+        self.assertEqual(len(self.test_obj['All'].get(node_ids=[]).ids), 0)
 
+        self.assertEqual(len(sel.times), 3)  # Number of timestamp (0.8, 1.0 and 1.2)
+        with self.assertRaises(SonataError): self.test_obj['All'].get(tstart=5.)  # tstart out of range
+        self.test_obj['All'].get(tstart=3., tstop=3.) # tstart should be < tstop
```

Comment:
Last nitpick, please change the comment here since now it is allowed tstart = stop.

Reply:
good catch ! Actually you should have raised here no ?

Reply:
I requested Nico to allow ```tstart == stop``` so that users can retrieve the data for one single timestamp (In fact I am using that in the circuit-faker benchmark test). So it should not raise error here.

Reply:
ahhh I did not recall correctly and thought you asked for the tstart == stop to raise ... 
Ok !